html/js/pedalboard.js

```diff
@@ -73,7 +73,7 @@ JqueryClass('pedalboard', {
             // The scale is the transform scale() css property that the pedalboard has
             baseScale: 0.5,
             // maxScale is the maximum zoom.
-            maxScale: 1,
+            maxScale: 1.2,
```

Comment:
I wanted to be able to zoom in on a plugin close enough, to easily operate the knobs this my fingers. On Surface, with GxAmplifier as example, 1.2 seemed most comfy

Reply:
changing the max zoom should be discussed and handled in a different PR, it might have side effects.
best to keep each PR as minimal as possible, 1 feature per PR basically